entity/src/main/java/dev/ikm/tinkar/entity/load/LoadEntitiesFromProtobufFile.java

```diff
@@ -77,6 +84,7 @@ public EntityCountSummary compute() {
         updateMessage("Importing Protobuf Data...");
 
         // Process Protobuf Entry
+        EntityService.get().beginLoadPhase();
```

Comment:
Do we also need to call these load lifecycle methods when we load the Spined Array store?

Reply:
The Spined Array Store looks like it is directly accessed as bytes on disk. As such, the usual method of calling a put API to add an Entity to the database is not used (at least as far as I can tell). The starter-data-builder and snomedct-transformer code can be updated in the future to make use of the lifecycle methods to reduce their execution time.